content/altinn-studio/app-creation/configuration/eformidling/_index.md

```diff
@@ -0,0 +1,186 @@
+---
+title: eFormidling
+description: Hvordan konfigurere eFormidling integrasjon for en app
+toc: true
+aliases:
+ - /altinn-studio/app-creation/configuration/eformidling
+weight: 400
+---
+
+## Aktivere integrasjon med eFormidling i applikasjonen din
+
+{{%notice info%}}
+For at applikasjonen din skal kunne sende instansdata videre til eFormidling må den referere til nugetversjon >= 4.2.0.
+Se hvordan du oppdaterer nugetreferanser for applikasjonen din [her](../update/#nuget-pakker).
+{{% /notice%}}
+
+Dersom man har behov for integrasjon med eFormidling i applikasjonen må dette aktiveres.
+
+I filen `appsettings.json` i mappen _App_ må følgende legges til i seksjonen _AppSettings_
+
+```json
+"EnableEFormidling":  true
+```
+
+I tillegg må det i samme fil opprettes en ny seksjon; _EFormidlingClientSettings_.
+Innholdet i kodesnutten nedenfor kan kopieres i sin helhet. 
+Denne setter opp url til integrasjonspunktet.
+Lenken peker på mocken som kan kjøres opp lokalt.
+Les mer om oppsettet av eFormidlings mocken [her](#-Kjøring-med-eFormidling-mock-lokalt).
+
+Når en app deployes til test eller produksjon vil denne verdien overskrives
+og peke mot integrasjonspunktet i Altinn Platform.
+
+
+```json
+"EFormidlingClientSettings": {
+   "BaseUrl": "http://localhost:9093/api/"
+ }
+```
+
+Dersom det ikke er ønskelig å teste integrasjonen med eFormidling når man kjører applikasjonen lokalt kan man overstyre
+denne konfigurasjonen i `appsettings.Development.json`. 
+Opprett _AppSettings_ seksjonen dersom den ikke finne og sett `EnableEFormidling` til false.
+
+```json
+"AppSettings": {
+    "EnableEFormidling": false
+}
+```
+
+## Konfigurere nøkkelverdier for eFormidling i applikasjonen din 
+
+Det kreves en del metadata om eFormidlingsforsendelsen og denne defineres i `applicationmetadata.json`.
+Filen finner du i repoet under mappen `App/config`.
+
+Opprett en ny seksjon `eFormidling` og fyll ut verdier for følgende parametre.
+
+| Id| Beskrivelse|
```

Comment:
Format table since markdown should be readable as markdown.
Tip: Use Alt + Shift + F in [this vscode extension](https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one).